bundle/schema/docs/bundle_descriptions.json

```diff
@@ -1521,6 +1521,43 @@
                   }
                 }
               },
+              "registered_models": {
```

Comment:
I think we can remove these changes. These changes are not reflected in the generated schema. Ideally these are meant to be autogenerated, but our autogeneration process for these docs is broken right now.

I have filed a followup to track this and we will automate this process soon.

Reply:
I can leave this in for now, and we can remove once the autogeneration works just so we have docs in the meantime

Reply:
Can you verify if these docs are visible in the `bundle schema` command. When I tried it out, they were not. My guess is skeleton structure might have an error. I am fine with keeping them if they work.

Note: We have an SOP for adding new docs, but this SOP is broken today and deletes existing handwritten documentation. https://github.com/databricks/cli/blob/main/bundle/schema/README.md

FYI I just autogenerated the skeleton structure expected for "registered_model". If this matches the docs should work and be present in the schema!
```
"registered_models": {
                "description": "",
                "additionalproperties": {
                  "description": "",
                  "properties": {
                    "catalog_name": {
                      "description": ""
                    },
                    "comment": {
                      "description": ""
                    },
                    "grants": {
                      "description": "",
                      "items": {
                        "description": "",
                        "properties": {
                          "principal": {
                            "description": ""
                          },
                          "privileges": {
                            "description": "",
                            "items": {
                              "description": ""
                            }
                          }
                        }
                      }
                    },
                    "name": {
                      "description": ""
                    },
                    "schema_name": {
                      "description": ""
                    },
                    "storage_location": {
                      "description": ""
                    }
                  }
                }
              }
```

Reply:
Ah I see, so the handwritten docs will be overwritten by this structure? If so, why are all the descriptions empty?

FWIW, my handwritten docs match the skeleton except the fields are not alphabetically sorted (is this a requirement?) and we also don't broadcast `storage_location` (generally not a field we want them to be modifying)

Reply:
>  handwritten docs will be overwritten by this structure? 
Yes. The descriptions for the API fields will be loaded from the openapi spec.

> are not alphabetically sorted (is this a requirement?)
That's not a requirement.

I'd expect the descriptions to be present in the `bundle schema` if the skeletons match, not sure why that's not happening.

The likely cause then for why this does not work is that our process to load docs is broken. Since this is not a blocker for getting UC models in, I'd recommend removing them for now and I'll take a note to add them back in as a followup when I fix this process.